quark/subproject.py

```diff
@@ -451,6 +483,19 @@ def write_excludes(fd):
         os.replace(quark_exclude_path, exclude_path)
 
 class SvnSubproject(Subproject):
+    @staticmethod
+    def get_version_control():
```

Comment:
Same as above.

Reply:
Changed in https://github.com/comelz/quark/pull/27/commits/ee6e41ed5393e010b8dd24de62bbfe021cc50f09